### DISCUSSION TITLE
Is there a fips-compliant build (or build process) available?

### DISCUSSION THREAD

**Discussion by dnovotny** [drakkan/sftpgo#1272] [Q&A] [answered]
We are looking to utilize SFTPGo, but need to verify that it is fips 140-2 compliant. I've read about using boringcrypto, or the redhat tool chain, to meet fips compliance needs, but am not sure how to go about that. We are currently using a version packaged via helm, but we can build our own image if we need to so that we can get fips compliance.
Is there any documentation on how to enable this that I can work from? 
Is adding "GOEXPERIMENT=boringcrypto" before the "go build..." Sufficient to support this?
Or are there any reasons this should not be attempted?
Thanks 
David

**Comment by drakkan** [comment, marked as answer]
You must build SFTPGo with a FIPS-enabled Go compiler and enable FIPS mode in your runtime environment. This way you can claim SFTPGo is using a FIPS-certified cryptographic module. Please check the [support policy](https://github.com/drakkan/sftpgo#support-policy). Thank you
